### ISSUE TITLE
Change overview diagram to formal UML Class / Domain diagram.

### ISSUE THREAD

**Issue by StanSwanborn** [S2-group/robot-runner#7]
The semantics of the arrows is not clear at all. Do they represent calls among modules? Do they represent interface Implementations (e.g., the ones between IRUNController and the other ...RunControllers below it)?

The same applies for the nodes of this diagram. Please, use a different representation for different entities (e.g., external Python scripts, config file, internal module, etc.). 

Here it could be nice also to show the robots, the necessary ROS topics and nodes, etc.

In summary: this diagram should be expanded in order to give a more detailed and comprehensive overview of what is happening in RobotRunner (and around it).
